test/parallel/test-debugger-invalid-json.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const startCLI = require('../common/debugger');
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const http = require('http');
+
+const host = '127.0.0.1';
+
+const testWhenBadRequest = () => {
```

Comment:
I don't think we need functions here, let's use blocks instead.
```suggestion
{
```